tests/Repository.cpp

```diff
@@ -137,6 +138,16 @@ void TestRepository::testShouldIgnore()
     EXPECT_THROW(repo->shouldIgnore(dir.absoluteFilePath(ignoredFileName)), Exception);
 }
 
+void TestRepository::testIdentitySetting()
+{
+    const QString name("name");
+    const QString email("email");
+    repo->init(testdir);
+    repo->setIdentity(Repository::Identity{name, email});
+    QCOMPARE(repo->identity().name, name);
+    QCOMPARE(repo->identity().email, email);
```

Comment:
Could this be changed to use a `Repository::Identity` object all the way? Something like:

```
const Repository::Identity id("name", "email");
repo->init(testdir);
repo->setIdentity(id);
QCOMPARE(repo->identity(), id);
```

Reply:
I added a `==` operator for `Repository::Identity` directly in the test. It's a bit awkward but allows for `QCOMPARE` between `Repository::Identity` instances.